script/grblcon.py

```diff
@@ -129,15 +160,19 @@ class GrblCon:
         'clear': '/0 Clear the screen',
         'close': '/0 Close the current grbl connection',
         'help': '/0 Display this dialog',
-        'home': '/0..4 (X/Y/Z/C) Homes the specified axis. Multiple axis may be provided (or none to home all)',
+        'home': ''.join([
+            '/0..4 (X/Y/Z/C) Homes specified axis.',
+            'Multiple axes may be provided (or none to home all)']),
```

Comment:
This seems needlessly complicated. What is your goal here? If you're trying to fit in a maximum width, just break the string up into multiple lines, without using `join` or a list:

```python
'home': ('/0..4 (X/Y/Z/C) Homes specified axis.'
         'Multiple axes may be provided (or none to home all)'),
```